addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -88,10 +88,9 @@ def _check_move_configuration(self, move):
         all_base_tags = self._get_l10n_in_base_tags()
         for line in move.invoice_line_ids.filtered(lambda line: line.display_type not in ('line_note', 'line_section', 'rounding')):
             if line.price_subtotal < 0:
-                # Line having a negative amount is not allowed.
-                if not move._l10n_in_edi_is_managing_invoice_negative_lines_allowed():
-                    raise ValidationError(_("Invoice lines having a negative amount are not allowed to generate the IRN. "
-                                  "Please create a credit note instead."))
+                move.message_post(
```

Comment:
Will you do a message_post for every line? (maybe most cases will be only 1 negative, but it is still quite possible)